docs/src/components/landing/BannerV2.js

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Container from '@mui/material/Container';
+import Button from '@mui/material/Button';
+import Link from 'docs/src/modules/components/Link';
+import Typography from '@mui/material/Typography';
+import KeyboardArrowRightRounded from '@mui/icons-material/KeyboardArrowRightRounded';
+
+function BannerV2(props) {
+  const { title, description, href, label, category, action, docs } = props;
```

Comment:
We probably don't need the `docs` prop on this component anymore since this was only to differentiate styles when re-using the same component between the landing page and the docs

Reply:
Ah, cool ⎯ that makes sense! I'll remove it! Let me know if keeping this component as `BannerV2` makes sense either 😬 I did it in such a way as to not delete anything that would break.

Reply:
I feel like if we're not using `Banner` anymore, we can get rid of it and just name the new component `Banner` - that is perfectly fine for me